examples/shaders/shaders_compute_gol.c

```diff
@@ -0,0 +1,147 @@
+/*******************************************************************************************
+*
+*   raylib [shaders] example - Compute shaders Conway's Game of Life
+*
+*   NOTE: This example requires raylib OpenGL 4.3 versions for compute shaders support,
+*
+*   NOTE: Shaders used in this example are #version 430 (OpenGL 4.3).
+*
+*   This example has been created using raylib 4.0 (www.raylib.com)
+*   raylib is licensed under an unmodified zlib/libpng license (View raylib.h for details)
+*
+*   Example contributed by Teddy Astie (@tsnake41)
+*
+*   Copyright (c) 2021 Teddy Astie (@tsnake41)
+*
+********************************************************************************************/
+
+#include <stdlib.h>
+
+#include "raylib.h"
+#include "rlgl.h"
+
+// IMPORTANT: This must match gol*.glsl GOL_WIDTH constant.
+//            This must be a multiple of 16 (check golLogic compute dispatch).
+#define GOL_WIDTH 768
+
+#define SSBO_SIZE (sizeof(unsigned int) * GOL_WIDTH * GOL_WIDTH)
+
+// Maximum amount of queued draw commands (squares draw from mouse down events).
+#define MAX_BUFFERED_TRANSFERTS 48
+
+struct GolUpdateCmd {
+  unsigned int x; // x coordinate of the gol command
+  unsigned int y; // y coordinate of the gol command
+  unsigned int w; // width of the filled zone
+  unsigned int enabled; // whether to enable or disable zone
+};
+
+struct GolUpdateSSBO {
+  unsigned int count;
+  struct GolUpdateCmd commands[MAX_BUFFERED_TRANSFERTS];
+};
+
+int main(void)

```

Comment:
Please, could you review the example and shaders to follow the [coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md)? Thanks!